src/main/java/webserver/HttpCookie.java

```diff
@@ -0,0 +1,57 @@
+package webserver;
+
+import webserver.httpmessage.HttpHeaders;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HttpCookie {
+    public static final String SESSION_KEY = "JSSESSIONID";
+    private final Map<String, String> values;
```

Comment:
```suggestion
    public static final String SESSION_KEY = "JSSESSIONID";
    
    private final Map<String, String> values;
```
상수와 상태 값은 구분 지어주면 좋을 것 같아요~